packages/cli/test/dev/integration-3.test.ts

```diff
@@ -70,27 +70,6 @@ test(
     await testPath(200, '/', /React App/m);
   })
 );
-/*
-test(
-  '[vercel dev] 05-gatsby',
-  testFixtureStdio('05-gatsby', async (testPath: any) => {
-    await testPath(200, '/', /Gatsby Default Starter/m);
-  })
-);
-*/
-test(
-  '[vercel dev] 06-gridsome',
```

Comment:
Removed this test since Gridsome hasn't had a new release in over 2 years, and the current version does not build on Node 18.

Reply:
👍

Reply:
Could it be pinned to Node.js 16 using package.json engines?

Reply:
The problem then is that our tests are using Node 14 still, so then `yarn install` fails when testing `vc dev`.

Reply:
Lets pin it to node 14 then 🙂